sqlx-mysql/src/types/mod.rs

```diff
@@ -21,7 +21,6 @@
 //! | `Ipv4Addr`                            | INET4 (MariaDB-only), VARCHAR, TEXT                  |
 //! | `Ipv6Addr`                            | INET6 (MariaDB-only), VARCHAR, TEXT                  |
 //! | [`MySqlTime`]                         | TIME (encode and decode full range)                  |
-//! | [`Duration`]                          | TIME (for decoding positive values only)             |
```

Comment:
A Duration type distinct from time::Duration and chrono::TimeDelta seems to not exist anymore

Reply:
This would be `std::time::Duration`. `Type` and `Decode` impls are defined in `mysql_time.rs`

Reply:
Thank you! i don't know how i missed it :sweat_smile:

Reply:
Fixed in 934f93c40affb5513dba21138e6502bf6ea5525e